climpred/tests/test_lead_time_resolutions.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+
+from climpred.prediction import compute_hindcast, compute_perfect_model
+
+
+@pytest.fixture()
+def daily_initialized():
+    init = pd.date_range('1990-01', '1990-03', freq='D')
+    lead = np.arange(5)
+    member = np.arange(5)
+    return xr.DataArray(
+        np.random.rand(len(init), len(lead), len(member)),
+        dims=['init', 'lead', 'member'],
+        coords=[init, lead, member],
+    )
+
+
+@pytest.fixture()
+def daily_reference():
+    time = pd.date_range('1990-01', '1990-03', freq='D')
+    return xr.DataArray(np.random.rand(len(time)), dims=['time'], coords=[time])
+
+
+@pytest.fixture()
+def monthly_initialized():
+    init = pd.date_range('1990-01', '1996-01', freq='MS')
+    lead = np.arange(20)
+    member = np.arange(5)
+    return xr.DataArray(
+        np.random.rand(len(init), len(lead), len(member)),
+        dims=['init', 'lead', 'member'],
+        coords=[init, lead, member],
+    )
+
+
+@pytest.fixture()
+def monthly_reference():
+    time = pd.date_range('1990-01', '1996-01', freq='MS')
+    return xr.DataArray(np.random.rand(len(time)), dims=['time'], coords=[time])
+
+
+def test_daily_resolution_hindcast(daily_initialized, daily_reference):
```

Comment:
All these tests could be parametrised over freq, units. Then later changes are easier. 
Not needed but nice.

Reply:
I see what you mean. I think it's nice to be explicit to help debug tests. It tends to be a headache when you have a bunch of parameterizations and are trying to figure out which is breaking and why.